membership-common/src/main/scala/com/gu/salesforce/Scalaforce.scala

```diff
@@ -127,60 +125,69 @@ abstract class Scalaforce(implicit ec: ExecutionContext) extends LazyLogging {
 
   object Query {
     def execute(query: String): Future[\/[String, JsValue]] = {
-      val path = s"services/data/v29.0/query?q=$query"
-      Timing.record(metrics, "Execute query") {
-        get(path)
-      }.map { response =>
-        val bodyString = response.body().string() // out here to make sure connection is closed in all cases
-        response.code() match {
-          case Status.OK => \/-(jsonParse(bodyString))
-          case code => -\/(s"SF004: Salesforce returned code $code for query: $query")
+      val path = s"services/data/v57.0/query?q=$query"
```

Comment:
No changes should be expected at all in the expected response from Salesforce when moving from v29.0 to v57.0.